docs/parameters.yaml

```diff
@@ -1268,6 +1268,8 @@ description: |+
   - "distanceAndLoad": Sorts caches according to both their distance and a calculated load. This is currently a placeholder,
     and returns the same ordering as "distance".
   - "random": Sorts caches randomly.
+  - "smart": Sorts caches by distance from the client, server IO load, object availability, and stochastically sort the result.
```

Comment:
How about something like "Sorts caches according to stochastically-generated weights that consider a combination of factors, including a cache's distance from the client, its IO load, and whether the cache already has the requested object."

On a side note, I don't love the term "smart" because I think this is the kind of thing Miron would have issues with. What about something like "balanced", "adaptive" or "dynamic"? Feel free to disagree to disagree with me on this, I'll defer to you.

Reply:
Naming is always hard! I like "adaptive" and really open to any suggestions